railties/lib/rails/generators/test_unit/authentication/templates/test/controllers/passwords_controller_test.rb.tt

```diff
@@ -0,0 +1,66 @@
+require "test_helper"
+
+class PasswordsControllerTest < ActionDispatch::IntegrationTest
+  test "new" do
+    get new_password_path
+    
+    assert_response :success
+  end
+
+  test "create" do
+    post passwords_path, params: { email_address: users(:one).email_address }
```

Comment:
Don't like hardcoding this to the stock fixtures, because they're so likely to change, and then it's a lot of busywork to update. What we could do instead is something like `User.take`, which will grab a random user to do this with.

Reply:
I rebased and updated the test according to your suggestion. Instead of relying on the `:one` fixture, I'm using `User.take` now instead 👍